### PR TITLE
chore: update ruby docs for using the sdk with preload_app

### PR DESCRIPTION
**Pull request by nsalamad**

### CODE REVIEW

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md [hunk 1]
```diff
@@ -44,4 +44,38 @@ options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_in
...
+```
+
+## Initializing the SDK in a Rails App Using Unicorn
+
+When using the `preload_app` configuration with Unicorn, the SDK needs to be initialized in the `after_work` block in the `config/unicorn.rb` file:
+
+```ruby
+after_fork do |server, worker|
+  options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_interval_ms: 1000, config_polling_interval_ms: 400)
+  Rails.configuration. = DevCycle::DVCClient.new(ENV['DVC_SERVER_SDK_KEY'], options, true)
```
Comment (suthar26) [inline docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:69]: should this be `Rails.configuration.dvc_client` instead?

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md [hunk 2]
```diff
@@ -44,4 +44,38 @@ options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_in
 dvc_client = DevCycle::DVCClient.new("dvc_server_token_hash", options, true)
```
Comment (suthar26) [inline docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:44]: do we say `token_hash` or just call it `sdk_token` ?

**Review by suthar26 — CHANGES_REQUESTED** [review]

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md [hunk 3]
```diff
@@ -44,4 +44,38 @@ options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_in
...
+**Step 2:** Add the following code to the `devcycle.rb` file:
+
+```ruby
+options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_interval_ms: 1000, config_polling_interval_ms: 4000)
+Rails.configuration.dvc_client = DevCycle::DVCClient.new(ENV['DVC_SERVER_SDK_KEY'], options, true)
+```
+
+## Initializing the SDK in a Rails App Using Unicorn
+
+When using the `preload_app` configuration with Unicorn, the SDK needs to be initialized in the `after_work` block in the `config/unicorn.rb` file:
```
Comment (suthar26) [inline docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:64]: what setting for `preload_app` or would both require the same?

**Review by suthar26 — CHANGES_REQUESTED** [review]

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md (hunk 3 above)
Comment (nsalamad) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:64]: it's just when it's set to true. Will clarify

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md (hunk 2 above)
Comment (nsalamad) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:44]: `sdk_token` is better. I'll make the change

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md (hunk 1 above)
Comment (nsalamad) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:69]: yep

**Review by suthar26 — APPROVED** [review]

**Pull request merged**